flow/genericactors.actor.h

```diff
@@ -2042,6 +2042,15 @@ class UnsafeWeakFutureReference {
 	Reference<UnsafeWeakFutureReferenceData> data;
 };
 
+// Call a lambda every <interval> seconds
+ACTOR template <typename Fn>
```

Comment:
There's already `recurring`

Reply:
As noted above, this PR includes another PR as its base, which I unfortunately just merged prior to seeing this comment, so I'll remove the new actor in this PR before finalizing.